contracts/mainnet/connectors/morpho-blue/helpers.sol

```diff
@@ -0,0 +1,190 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {Id, IMorpho, MarketParams, Position, Market} from "./interfaces/IMorpho.sol";
+import "../../common/stores.sol";
+import "../../common/basic.sol";
+import "../../common/interfaces.sol";
+import {MorphoBalancesLib} from "./libraries/periphery/MorphoBalancesLib.sol";
+import {MorphoLib} from "./libraries/periphery/MorphoLib.sol";
+import {UtilsLib} from "./libraries/UtilsLib.sol";
+import {MarketParamsLib} from "./libraries/MarketParamsLib.sol";
+import {SharesMathLib} from "./libraries/SharesMathLib.sol";
+
+abstract contract Helpers is Stores, Basic {
+    using MorphoBalancesLib for IMorpho;
+    using MorphoLib for IMorpho;
+    using MarketParamsLib for MarketParams;
+    using UtilsLib for uint256;
+    using SharesMathLib for uint256;
+
+    IMorpho public constant MORPHO_BLUE =
+        IMorpho(0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb);
+
+    uint256 internal constant MARKET_PARAMS_BYTES_LENGTH = 5 * 32;
+
+    /// @dev The number of virtual assets of 1 enforces a conversion rate between shares and assets when a market is
+    /// empty.
+    uint256 internal constant VIRTUAL_ASSETS = 1;
+
+    /// @dev The number of virtual shares has been chosen low enough to prevent overflows, and high enough to ensure
+    /// high precision computations.
+    uint256 internal constant VIRTUAL_SHARES = 1e6;
```

Comment:
These constants are redundant with the constants defined in `SharesMathLib`

Reply:
Yess, they were previously being used. Removed them